src/api/app/models/bs_request_action_maintenance_release.rb

```diff
@@ -59,8 +59,9 @@ def per_request_cleanup(opts)
 
       next if cleaned_projects[sprj]
 
-      # cleanup published binaries to save disk space on ftp server and mirrors
-      Backend::Api::Build::Project.wipe_published_locked(sprj)
+      # Waiting 2 days before removing published repositories for accepted release requests,
+      # gives some margin to the automated maintenance tests to be finished.
+      PublishedRepositoriesCleanupJob.set(wait: 2.days).perform_later(sprj)
```

Comment:
:thought_balloon: What about moving the `2.days` into a constant?

Reply:
The value has been moved to options.yml and use seconds instead of days, in case some instance needs to configure a shorter delay (minutes, hours...).

Reply:
:+1: